src/components/ContentNode/TabNavigator.vue

```diff
@@ -71,6 +71,21 @@ export default {
       currentTitle: this.titles[0],
     };
   },
+  watch: {
+    titles(newVal, oldVal) {
+      if (newVal.length < oldVal.length) {
+        // set to the first tab if selected tab was removed
+        if (!newVal.includes(this.currentTitle)) {
+          // eslint-disable-next-line prefer-destructuring
+          this.currentTitle = newVal[0];
```

Comment:
Instead of disabling ESLint, you could:

```suggestion
          const [firstTitle] = newVal;
          this.currentTitle = firstTitle;
```